build.sbt

```diff
@@ -74,15 +74,17 @@ lazy val root = Project("scalac-scoverage", file("."))
 
 lazy val runtime = CrossProject("scalac-scoverage-runtime", file("scalac-scoverage-runtime"))(JVMPlatform, JSPlatform)
     .crossType(CrossType.Full)
+    .withoutSuffixFor(JVMPlatform)
     .settings(name := "scalac-scoverage-runtime")
     .settings(appSettings: _*)
     .settings(
       libraryDependencies += "org.scalatest" %%% "scalatest" % ScalatestVersion % Test
     )
     .jvmSettings(
-      fork in Test := true
+      Test / fork := true
     )
     .jsSettings(
+      crossVersion := CrossVersion.fullWith("sjs" + scalaJSVersion.substring(0, 3) + "_", ""),
```

Comment:
Cool, so I just tested this in your sbt plugin pr and seems to work. However, just for context, is this standard in the Scala JS ecosystem? In other artifacts I see `_sjs1_`, or `_sjs0.5`, but not really the full `_sjs1.5`. I'm not sure how many projects would actually directly use this artifact apart from the plugins, but this would force them to use Scala JS 1.5.x correct?

Reply:
I chatted with the ScalaJS folks and got this clarified. We are going to only grab the 1. I pushed a commit on top of yours to push this forward.

Reply:
Alright 👍